core/common/connector-core/README.md

```diff
@@ -4,15 +4,4 @@ Extension that registers default and core services as default implementations, h
 
 ## Configuration settings
```

Comment:
please remove the section altogether, the link for the autodoc is already listed in the documentation site.